app/_hub/nonamesecurity/nonamesecurity-kongprevention/overview/_index.md

```diff
@@ -0,0 +1,125 @@
+
+## How it works
+
+Prevention integrations enable Noname to automatically block, or you to manually block, suspicious or dangerous runtime traffic to your APIs. The integration enables Noname to perform the blocking operation in an external system, such as an AWS API Gateway or an Azure API Manager at the same time it noname also informs Kong to block the ip address or OAuth JWT token associated with a known anomalous actor be it a bot-net or a hacker tool or a QA test tool. When blocking, you can specify a time period for the block; the external system manages the block and the removal of the block.  
+
+The nonamesecurity plugin is a prerequisite when using the noname-prevention plugin.
+
+The Noname Security Kong plugin is available as a lua rocks module via the noname integrations admin console.  Noname Security prevention integration for Kong uses a noname provided custom lua rocks plugin to [identify and block attackers](https://docs.nonamesecurity.com/docs/kong-prevention) that were found using the Noname anomaly detection plugin called nonamesecurity. 
+
+### Performance Benchmarks 
+[Kong Enterprise 2.8.1.1 with Noname 3.1](https://docs.nonamesecurity.com/v320/docs/kong-performance-results)
+
+
+## How to install
+
+### The Noname blocking and prevention plugin for Kong 
+
+[Noname Security Kong Prevention install and configuration Documentation](https://docs.nonamesecurity.com/docs/kong-prevention)
+
+1. Create the Integration Profile
+
+In the Noname UI, navigate to Settings > Integrations > Prevention.
+Select Add Integration and select the Kong tile to create an integration profile.
+Download the Zip file, and copy it to your Kong machine.
+Select Next.
+Enter the Kong Admin API URL.
+Enter key-value sets for the Authentication headers to enable noname to authenticate with Kong.
+Select Next.
+Provide an alias for the integration.
+Select Finish to save the integration.
+
+2. Install the LuaRocks module
+
+Upload and activate the noname prevention plugin to kong using the following steps: In your Kong machine CLI shell Navigate to the location of the copied Zip file and unzip the file.
+Run the following command to install the plugin:
+
+
+```shell
+luarocks make
+
+```
+
+3. Update the Kong plugins configuration environment variable 
+
+Add the noname-prevention plugin to the kong configuration file (usually /etc/kong/kong.conf).
+
+    Open the file, find the plugins, and add noname-prevention. For example:
+    
+```yaml
+plugins = bundled,noname-prevention
+```
+
+4. Restart Kong
+
+After LuaRocks is installed, restart Kong before enabling the plugin
+
+```shell
+kong restart
+```
+
+### Install via a Dockerfile to create a custom kong docker image with both of the noname kong plugins preinstalled
```

Comment:
```suggestion
### Install via a Dockerfile

Instead of manually installing with `kong.conf`, you can also use a Dockerfile to create a custom kong docker image with both of the Noname Kong plugins preinstalled:
```